validator/client/runner.go

```diff
@@ -172,3 +182,33 @@ func handleAssignmentError(err error, slot uint64) {
 		log.WithField("error", err).Error("Failed to update assignments")
 	}
 }
+
+func handleAccountsChanged(ctx context.Context, v Validator, accountsChangedChan chan struct{}) {
```

Comment:
Any test for this function?

Reply:
Good idea. This code is executed when testing `WaitForActivation`, but it is beneficial to test this function in isolation.

Reply:
After some consideration I am in favor of postponing these tests until dynamic key reloading for the remote keymanager is ready. This will allow to define `SubscribeAccountChanges` as an interface function and make testing much easier.

Reply:
I'm not in favor of postponing tests on such mission critical functionality. 

If something is blocking tests for this PR, wait until that issue has been resolved then add tests before proposing these changes.